regression/goto-instrument/insert-final-assert-false1/main.c

```diff
@@ -3,6 +3,12 @@
 int main()
 {
   size_t n;
+  // alloc size is arbitrary but no larger than what CBMC can represent
+  size_t pointer_width = sizeof(void *) * 8;
+  size_t object_bits = 8; // by default
+  size_t cbmc_max_alloc_size = ((size_t)1 << (pointer_width - object_bits)) - 1;
+  __CPROVER_assume(n <= cbmc_max_alloc_size);
```

Comment:
May be worth adding this as an exposed constant `__CPROVER_max_alloc_size`? Seems to me this might come up fairly regularly in user scenarios as well, and it’d probably be a bit of a PITA to keep redefining this.